islandora_bookmark.install

```diff
@@ -91,6 +97,18 @@ function islandora_bookmark_update_7100(&$sandbox) {
 }
 
 /**
+ * Adds a description field to bookmarks.
+ */
+function islandora_bookmark_update_7101(&$sandbox) {
+  db_add_field('islandora_bookmark_list_names', 'descrip', array(
+    'description' => 'The supplied description of the list',
+    'type' => 'varchar',
+    'length' => 255,
+    'default' => "Bookmark List",
```

Comment:
Should probably make this the empty string due to the retroactive nature of db_add_field.